releasenotes/notes/0.9/improvements-in-DAG-drawer-73283f18ba5ba937.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The DAG drawer was enhance in the context of Jupyter notebooks. The parameter filename is now
+    optional and the function will show the result in the notebook.
+  - |
+    The DAG drawer can be acces through the `DAGCircuit.draw(...)` method. This is equivalent to
```

Comment:
```suggestion
    The DAG drawer can be accessed through the `DAGCircuit.draw(...)` method. This is equivalent to
```

Reply:
Minor style thing, not a blocker:

The form I was using in the rest of the release notes was:

```
the `qiskit.dagciruit.DAGCircuit` class has a new method `draw()`. This is equivalent to
running `qiskit.visualization.dag_drawer()` on a given `DAGCircuit` object.
```